Missionframework/modules/06_arsenal/fnc/fn_arsenal_postInit.sqf

```diff
@@ -0,0 +1,40 @@
+/*
+    KPLIB_fnc_arsenal_postInit
+
+    File: fn_arsenal_postInit.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-11-12
+    Last Update: 2018-11-25
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        The postInit function of a module takes care of starting/executing the modules functions or scripts.
+        Basically it starts/initializes the module functionality to make all provided features usable.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Module postInit finished [BOOL]
+*/
+
+if (isServer) then {
+    diag_log format ["[KP LIBERATION] [%1] [POST] [ARSENAL] Module initializing...", diag_tickTime];
+
+    // Load arsenal
+    [] call KPLIB_fnc_arsenal_fillArsenal;
+};
+
+// Setup of actions available to players
+[] call KPLIB_fnc_arsenal_setupPlayerActions;
+
+// Add event handler for gear check
+if (hasInterface) then {
```

Comment:
```suggestion
if (hasInterface) then {
    [missionNamespace, "arsenalOpened", {player setVariable ["KPLIB_backpack", backpack player]}] call BIS_fnc_addScriptedEventHandler;
```

Reply:
This provides a saving of the current backpack of the player.
The benefit is, that we keep the old gearCheack feature, that it won't blacklist your packed mortar/HMG/GMG backpack, if you enter the arsenal with it. This could be the case if you've disassembled a buyed static weapon and wanted to get some more ammo or something else from the arsenal.
This adjustment would need one test afterwards, of course.

Reply:
Should be done for ACE arsenal too?!

Reply:
Yeah, makes sense.